scripts/mina-local-network/mina-local-network.sh

```diff
@@ -554,6 +561,18 @@ if ${UPDATE_GENESIS_TIMESTAMP}; then
   fi
 fi
 
+if [ ! -z "${OVERRIDE_SLOT_TIME_MS}" ]; then
+  echo 'Modifying configuration to override slot time'
+  if test -f "${CONFIG}"; then
+    printf "\n"
+
+    tmp=$(mktemp)
+  else
+    reset-genesis-ledger ${LEDGER_FOLDER} ${CONFIG}
+  fi
+  jq ".proof.block_window_duration_ms=${OVERRIDE_SLOT_TIME_MS}" ${CONFIG} >"$tmp" && mv -f "$tmp" ${CONFIG}
```

Comment:
```suggestion
    tmp=$(mktemp)
    jq ".proof.block_window_duration_ms=${OVERRIDE_SLOT_TIME_MS}" ${CONFIG} >"$tmp" && mv -f "$tmp" ${CONFIG}
  else
    reset-genesis-ledger ${LEDGER_FOLDER} ${CONFIG}
  fi
```

Reply:
Since `tmp` is declared conditionally.

Reply:
Perhaps we don't even need the `reset-genesis-ledger` invocation (branch) for slot time overriding?